data-raw/otu_mini_continuous.R

```diff
@@ -0,0 +1,85 @@
+## code to prepare `otu_mini` dataset
+set.seed(2019)
+outcome_colname <- "Otu00001"
+kfolds <- 2
+otu_mini <- otu_small[, 1:4]
+# usethis::use_data(otu_mini, overwrite = TRUE)
+
+test_hyperparams <- structure(list(
```

Comment:
Is there a reason you're defining `test_hyperparams` here instead of letting `run_ml()` use the hyperparams function?

Reply:
We should probably just get rid of `test_hyperparams`, they're leftover from the time before we had functions to get default hyperparameters dynamically.

Reply:
I think 1) it takes longer with all the hyperparameters, and 2) I was getting warnings, but now that we're going to fix that it's probably okay to use all of them. Think we should fix that here or in a separate branch?